packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -2188,6 +2288,10 @@ export class IntervalCollection<TInterval extends ISerializableInterval> extends
 					newStart,
 					interval.start.refType,
 					op,
+					interval.stickiness !== undefined &&
```

Comment:
logic similar to this shows up in a couple places. i'd pull it out into a helper.